src/www/services_unbound_advanced.php

```diff
@@ -338,21 +338,21 @@
                           <select id="log_verbosity" name="log_verbosity" class="selectpicker">
 <?php
                           for ($level = 0; $level <= 5; $level++):?>
-                            <option value="<?= $level; ?>" <?=$pconfig['log_verbosity'] == $level ? "selected=\"selected\"" : ""; ?>>
-                              <?=gettext("Level");?> <?=$level; ?>
+                            <option value="<?= $level; ?>" <?=$pconfig['log_verbosity'] == $level ? 'selected="selected"' : ''; ?>>
+                              <?= sprintf(gettext("Level %s"), $level) ?>
                             </option>
 <?php
                           endfor;?>
                           </select>
                           <div class="hidden" for="help_for_log_verbosity">
-                            <?=gettext("Select the log verbosity.");?>
+                            <?= gettext("Select the log verbosity.") ?>
                           </div>
                         </td>
                       </tr>
                       <tr>
                         <td></td>
                         <td>
-                          <input type="submit" name="Save" class="btn btn-primary" id="save" value="Save" />
+                          <button type="submit" name="Save" class="btn btn-primary" id="save" value="Save"><?= gettext("Save") ?></button>
```

Comment:
@fabianfrz shouldn't this be a normal input/submit? the submit is part of the <form/> tag.